src/NGPL.xml

```diff
@@ -0,0 +1,95 @@
+<spdx name="Nethack General Public License" identifier="NGPL" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/NGPL</url>
+  </urls>
+  <license>
+    <title>
+      <p>NETHACK GENERAL PUBLIC LICENSE</p>
+    </title>
+    <copyright>
+      <p>(Copyright 1989 M. Stephenson)</p>
+    </copyright>
+    <optional>
+      <br/>(Based on the BISON general public license, copyright 1988 Richard M. Stallman)
+      <br/>
+    </optional>
+    <body>
+      <p>Everyone is permitted to copy and distribute verbatim copies of this license, but changing it is not
+         allowed. You can also use this wording to make the terms for other programs.</p>
+      <p>The license agreements of most software companies keep you at the mercy of those companies. By contrast,
+         our general public license is intended to give everyone the right to share NetHack. To make sure that
+         you get the rights we want you to have, we need to make restrictions that forbid anyone to deny you
+         these rights or to ask you to surrender the rights. Hence this license agreement.</p>
+      <p>Specifically, we want to make sure that you have the right to give away copies of NetHack, that you
+         receive source code or else can get it if you want it, that you can change NetHack or use pieces of it
+         in new free programs, and that you know you can do these things.</p>
+      <p>To make sure that everyone has such rights, we have to forbid you to deprive anyone else of these rights.
+         For example, if you distribute copies of NetHack, you must give the recipients all the rights that you
+         have. You must make sure that they, too, receive or can get the source code. And you must tell them
+         their rights.</p>
+      <p>Also, for our own protection, we must make certain that everyone finds out that there is no warranty for
+         NetHack. If NetHack is modified by someone else and passed on, we want its recipients to know that
+         what they have is not what we distributed.</p>
+      <p>Therefore we (Mike Stephenson and other holders of NetHack copyrights) make the following terms which say
+         what you must do to be allowed to distribute or change NetHack.</p>
+      <p>COPYING POLICIES</p>
+      <p>You may copy and distribute verbatim copies of NetHack source code as you receive it, in any medium,
+         provided that you keep intact the notices on all files that refer to copyrights, to this License
+         Agreement, and to the absence of any warranty; and give any other recipients of the NetHack program a
+         copy of this License Agreement along with the program.</p>
+      <p>You may modify your copy or copies of NetHack or any portion of it, and copy and distribute such
+         modifications under the terms of Paragraph 1 above (including distributing this License Agreement),
+         provided that you also do the following:</p>
+      <list>
```

Comment:
extra list tag, but see comment above - may want to just add higher level list above